dashboard-frontend/mocks/state.js

```diff
@@ -0,0 +1,286 @@
+import Select from './../src/pages/NewStudyPage/components/form/inputs/Select';
+import Text from './../src/pages/NewStudyPage/components/form/inputs/Text';
+
+const state = [
+  {
+    create_study: [
+      {
+        id: 'name',
+        name: 'name',
+        type: 'text',
+        component: Text,
+        label: 'Give your study a name',
+        helper_text: 'E.g example-fly-conf',
+        options: undefined,
+        value: '',
+      },
+    ],
+    general: [
+      {
+        id: 'objective',
+        name: 'objective',
+        type: 'select',
+        component: Select,
+        label: 'Objective',
+        helper_text: undefined,
+        options: [
```

Comment:
I don't know if we need to support _all_ the options for objectives and optimization_goals, but this is great for now!